recipe/conda_build_config.yaml

```diff
@@ -53,10 +61,14 @@ cuda_compiler:
   - None
   - nvcc                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
   - nvcc                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - cuda-nvcc                  # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - cuda-nvcc                  # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
 cuda_compiler_version:
   - None
   - 11.2                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
   - 11.8                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 12.0                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 12.4                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

Comment:
Are packages built for 12.0 expected to always successfully build with 12.4? Shouldn't we separate the closing of the 12.0 migration from adding 12.4...?

Reply:
I remember that in some feedstocks I had to add some code to explicitly use 12.0 to compile. This code should work with 12.4 but as it is not tested, I would feel better if we had a migrator first.